controlsfx/src/main/java/org/controlsfx/control/CheckBitSetModelBase.java

```diff
@@ -319,13 +319,13 @@ public BitSetReadOnlyUnbackedObservableList(BitSet bitset) {
                 return -1;
             }
 
-            if (index == (lastGetIndex + 1) && lastGetValue < itemCount) {
+            if (index == (lastGetIndex + 1) && index < itemCount) {

```

Comment:
index is already tested line 318. The method return if index is superior or equals to itemCount.
So at this step the condition will always be true. Perhaps it could be removed?